composer.json

```diff
@@ -66,12 +66,13 @@
         "symfony/dependency-injection": "~2.8|~3.0",
         "authorizenet/authorizenet": "~1.8",
         "sofort/sofortlib-php": "^3.0",
-        "paypal/rest-api-sdk-php" : "0.5.*",
+        "paypal/rest-api-sdk-php" : "~1.0",
         "klarna/checkout": "~1|~2.0",
         "fp/klarna-invoice": "0.1.*",
         "stripe/stripe-php": "~2.0|~3.0",
         "php-http/guzzle6-adapter": "^1.0",
-        "ext-soap": "*"
+        "ext-soap": "*",
+        "paypal/sdk-core-php": "~3.0"
```

Comment:
could you add it just after the rest-api-sdk-php so the paypal/xxx packages stay together